src/couchapps/WMStats/_attachments/js/ViewModels/WMStats.ViewModel.js

```diff
@@ -421,15 +421,16 @@ WMStats.ViewModel = (function (){
             }
             
             var requestInfo = reqMgrRequest.getData();
+            var getProperyByTask = WMStats.Requests.getPropertyByTask;
```

Comment:
Haaaa, if there is no typo, it's not your PR... but I give you this, it's consistent at least :-)
Can you update it in all the places please?